akka-contrib/docs/receive-pipeline.rst

```diff
@@ -16,33 +16,46 @@ Some Possible Use Cases
 
 Interceptors
 ------------
-So how does an interceptor look? Well, Interceptors are defined by decorator functions
-of type :class:`Receive => Receive`, where it gets the inner :class:`Receive` by parameter and
-returns a new :class:`Receive` with the decoration applied.
-Most of the times your decorators will be defined as a regular :class:`Receive` with cases
-for the messages of your interest and at some point delegate on the inner :class:`Receive`
-you get by parameter. We will talk about ignored and unhandled messages later.
+Multiple interceptors can be added to actors that mixin the :class:`ReceivePipeline` trait. 
+These interceptors can be thought of as layers: each outer interceptor advising the 
+next interceptor in, and so on, until the last or innermost interceptor which advises the 
+actor's :class:`Receive`. 
+
+The first or outermost interceptor receives messages sent to the actor. 
+
+For each message received by an interceptor, the interceptor will typically perform some 
+processing based on the message and decide whether 
+or not to pass the received message (or some other message) to the next interceptor.
+
+So what does an interceptor look like? 
```

Comment:
Maybe remove this sentence, does not fit the flow of the doc now with extended docs.